### DISCUSSION TITLE
How are classifiers populated?

### DISCUSSION THREAD

**Discussion by pawamoy** [pdm-project/pdm#453] [Q&A] [answered]
My project [mkdocs-coverage](https://github.com/pawamoy/mkdocs-coverage) is compatible with Python 3.6, yet PDM seems to put only the Python 3, Python 3.7, Python 3.8 and Python 3.9 classifiers in the wheel metadata: https://pypi.org/project/mkdocs-coverage/

How are the classifiers computed?

**Comment by frostming** [comment, marked as answer]
Because `3.6.0` is excluded out and pdm-pep517 thought it was not compatible with python 3.6

**Reply by pawamoy** [reply]
Should I open an issue on pdm-pep517 then?

**Reply by frostming** [reply]
Sure thing
